Bottom: b8f1ecec09004dd7f26d6892339dc6fd1ce944ed
Top:    bdede016107e7887162f7a1d60004733141635c3
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-05-14 11:59:58 -0500

Refresh of TESTING--ASDF

---
diff --git a/Vagrantfile b/Vagrantfile
index 9137e60..78f9816 100644
--- a/Vagrantfile
+++ b/Vagrantfile
@@ -1,21 +1,45 @@
 Vagrant.configure("2") do |config|
-    config.vm.box = "daedalus"
-    config.vm.box_url = "vmdaedalus/vmdaedalus-virtualbox.box"
-    config.vm.define "vmdaedalus.novatech-llc.com"
-    config.vm.hostname = "vmdaedalus.novatech-llc.com"
-    config.vm.provider "virtualbox" do |vb|
-        vb.name = "vmdaedalus"
+
+    config.vm.define "vmdaedalus.novatech-llc.com", primary: true do |main|
+        main.vm.box = "daedalus"
+        main.vm.box_url = "vmdaedalus/vmdaedalus-virtualbox.box"
+        main.vm.hostname = "vmdaedalus.novatech-llc.com"
+        main.vm.provider "virtualbox" do |vb|
+            vb.name = "vmdaedalus"
+        end
+    #    config.vm.network "public_network", ip: "192.168.0.100", netmask:"255.255.0.0", bridge: "enp0s31f6"
+        main.vm.network "private_network", ip: "192.168.0.100", netmask:"255.255.0.0"
+        main.vm.synced_folder "/mnt/vm/bacula-restores",
+            "/mnt/bacula-restores",
+            id: "bacula-restores",
+            owner: "root",
+            group: "root",
+            mount_options: ["ro"]
+    end
+
+    config.vm.define "buildbot-worker.novatech-llc.com" do |worker|
+        worker.vm.box = "centos/atomic-host"
+        worker.vm.hostname = "buildbot-worker.novatech-llc.com"
+        worker.vm.provider "virtualbox" do |vb|
+            vb.name = "bbworker"
+        end
+
     end
-#    config.vm.network "public_network", ip: "192.168.0.100", netmask:"255.255.0.0", bridge: "enp0s31f6"
-    config.vm.network "private_network", ip: "192.168.0.100", netmask:"255.255.0.0"
-    config.vm.synced_folder "vmdaedalus/bacula-restores", "/tmp/bacula-restores", type: "rsync"
 
     config.vm.provision "ansible" do |ansible|
         ansible.verbose = "v"
         ansible.playbook = "ansible-playbook/site.yml"
         ansible.groups = {
             "bacula" => ["vmdaedalus.novatech-llc.com"],
-            "docker-hosts" => ["vmdaedalus.novatech-llc.com"]
+            "docker-hosts" => ["vmdaedalus.novatech-llc.com"],
+            "buildbot" => [
+                "vmdaedalus.novatech-llc.com",
+                "buildbot-worker.novatech-llc.com"
+            ],
+            "docker-hosts:vars" => {
+                "openssl_image_repo" => "https://github.com/AndrewCooper/docker-openssl.git",
+                "openldap_image_repo" => "https://github.com/AndrewCooper/docker-openldap.git"
+            }
         }
         # ansible.tags = []
         # ansible.skip_tags = ["restore"]
diff --git a/vmdaedalus/vmdaedalus.json b/vmdaedalus/vmdaedalus.json
index 703e2b6..2703db8 100644
--- a/vmdaedalus/vmdaedalus.json
+++ b/vmdaedalus/vmdaedalus.json
@@ -23,7 +23,8 @@
             ["modifyvm", "{{.Name}}", "--memory", "512"],
             ["modifyvm", "{{.Name}}", "--usb", "off"],
             ["modifyvm", "{{.Name}}", "--audio", "none"]
-        ]
+        ],
+        "disk_size": "120000"
     }],
     "provisioners": [{
         "type": "shell",
